src/core/js/scriptLoader.js

```diff
@@ -8,6 +8,18 @@
         return false;
     })();
 
+    // Inject loading spinner into wrapper
+    domReady(function() {
+        var spinner = document.createElement("div");
+        var attr = document.createAttribute("class");
+        attr.value = "loading";
+        spinner.attributes.setNamedItem(attr);
```

Comment:
In Firefox I get a warning that `document.createAttribute()` is deprecated. Would it also just be simpler to condense the three lines into one?
```javascript
spinner.className = "loading";
```